src/main/java/com/mattmalec/pterodactyl4j/client/entities/Utilization.java

```diff
@@ -42,15 +54,37 @@ default String getUptimeFormatted() {
 			return String.format("%.0fh %.0fm %.0fs", hours, minutes, seconds);
 	}
 
+	/**
+	 * Get the current ram usage
+	 * @return the current ram usage in MB
+	 */
 	long getMemory();
+
+	/**
+	 * Get the current memory formatted in your desired data type such as {@link DataType#MB}
+	 * @param dataType the size of the data to use such as a GB/MB etc
+	 * @return the formatted string representation of the currently used memory
+	 */
 	default String getMemoryFormatted(DataType dataType) {
 		return String.format("%.2f %s", getMemory() / (dataType.getMbValue() * Math.pow(2, 20)), dataType.name());
 	}
+	/**
+	 * Get the current disk space usage
```

Comment:
This stuff is very recursive. We also don't want to start our doc with "Get" if weren't not actually running any logic to actually "get" the value, we're just returning data that's already there.
```suggestion
	 * The raw amount of disk space currently being used by this Server.
	 * @return The raw amount of disk space, usually in {@link DataType#MB} form
```